### PR TITLE
min_samples_in_file kwarg for raw_to_rawnc

### LINKED ISSUE #16: Should the length threshold for skipping writing nc files be an argument?
See: https://github.com/jklymak/pyglider/blob/92bfa4e1e480cf0c2f7f3398719a6a16cc8eb0ac/pyglider/seaexplorer.py#L136

It occured to me that hardcoding that value could be problematic (e.g. for very shallow dives with `.sub.` realtime data). I don't have the experience to be confident adding as an arg to `raw_to_rawnc()` though.

### PR DESCRIPTION
**Pull request by richardsc**

### CODE REVIEW

File: pyglider/seaexplorer.py [hunk 1]
```diff
@@ -61,6 +61,10 @@ def raw_to_rawnc(indir, outdir, deploymentyaml, incremental=True):
         If *True* (default), only netcdf files that are older than the
         binary files are re-parsed.
 
+    min_samples_in_file : int
+        Minimum number of samples in a yo to trigger writing a netcdf file.
```
Comment (jklymak) [inline pyglider/seaexplorer.py:65]:
```suggestion
        Minimum number of samples in a raw file to trigger writing a netcdf file.
```

**Comment by jklymak** [comment]
I think if you rebase on master, this will cause CI to run?

**Comment by richardsc** [comment]
Hm, rebasing is not a workflow that I'm overly familiar with TBH -- wouldn't it require merging the PR first?

**Comment by jklymak** [comment]
Not usually.  Assuming you call the GitHub branch `upstream` you do

```
% git fetch upstream   # this gets the new commits on upstream/master since you last fetched
% git rebase upstream/master    # this applies your commits on top of the new commits.
% git push origin master --force  # this pushes your changes to your branch that this PR is on
```

If you are lucky that is all there is to it.  If there have been conflicting changes, then you need to do edit them out by hand and do a song and dance that git tries to explain to you.  

BTW, you may want to get in the habit of naming your branches before submitting them as PRs.  

But... no need to do all this for this PR if you dont' want to.  I was just surprised that CI didn't run.

**Comment by richardsc** [comment]
Ok, thanks. I'm familiar with doing `git merge` and sorting out any associated conflicts, but it's the history changes that have always kept me from freely doing `git rebase` in my regular workflow.

And I agree about doing PRs from named branches -- that was my error as I had just merged the other PR into `master` and then  started working on it without thinking (I blame it being late at night on the East Coast ... 😄 ).

**Comment by richardsc** [comment]
Ok, I just rebase and pushed.

**Comment by jklymak** [comment]
OK< and CI did still didn't run so I have to fix that (actually I think I know the problem).  Thanks for doing that!

**Pull request merged**